### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by Raklyon**

### CODE REVIEW

**Comment by Raklyon** [comment]
Fixes issue #1 and #8

File: back/src/main/java/back/back/Cercle.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	public void setCentre(int[] centre) {
+		this.centre = centre;
+	}
+
+	public double getRayon() {
+		return rayon;
+	}
+
+	public void setRatyon(int rayon) {
```
Comment (Marsobad) [inline back/src/main/java/back/back/Cercle.java]: setRayon

File: back/src/main/java/back/back/Dessin.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package back.back;
+
+import java.util.HashMap;
+
+public class Dessin {
+	private static final Exception WrongIdException = null;
+	// On utilise une ArrayList pour pouvoir profiter de l'aspect dynamique de cette structure de données.
```
Comment (Marsobad) [inline back/src/main/java/back/back/Dessin.java]: hashmap

File: back/src/main/java/back/back/Dessin.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	private HashMap<Integer, Forme> formes = new HashMap<Integer, Forme>();
+
+
+	public void addForme(Forme newForme) throws Exception {
+		synchronized(formes) {
+			if(formes.containsKey(newForme.getId())) {
+				throw WrongIdException;
```
Comment (Marsobad) [inline back/src/main/java/back/back/Dessin.java]: Pour plus tard , gérer l'exception

**Pull request merged**